packages/multi-entry/test/test.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-bitwise */
+
+import test from 'ava';
```

Comment:
Most of our tests have been using CommonJS, should we use it here too?

Reply:
ava comes with baked in support for es6 via babel. eventually I'd like to see esm in tests and we have a stated goal of getting all source on typescript and esm. this test is just ahead of the curve.